README.md

```diff
@@ -17,7 +17,7 @@
 |:---|:---|:---|:---|
 | Ansible |  [ansible](./topics/ansible/)| 📖 [docs.ansible.com](https://docs.ansible.com/)|✔️ [ansible-helloworld.sh](./topics/ansible/hello-world/ansible-helloworld.sh)|
 | Docker |  [docker](./topics/docker/)| 📖 [docs.docker.com](https://docs.docker.com/)|✔️ [docker-helloworld.sh](./topics/docker/docker-helloworld.sh)|
-| Kubernetes (k8s) | ✔️ [k8s](./topics/k8s/)| 📖 [kubernetes.io/docs](https://kubernetes.io/docs/home/) |✔️ [k8s-helloworld.sh](./topics/k8s/k8s-helloworld.sh)|
+| Kubernetes (k8s) |  [k8s](./topics/k8s/)| 📖 [kubernetes.io/docs](https://kubernetes.io/docs/home/) |✔️ [k8s-helloworld.sh](./topics/k8s/k8s-helloworld.sh)|

```

Comment:
Thanks @dcompanykrish 
Similar comment with https://github.com/tungbq/devops-basic/pull/286#pullrequestreview-1633343039
In 1 PR we should only cover 1 issue. Please keep this line as is. Only change the greeting in this PR.